Source/Core/Matrix3.js

```diff
@@ -303,6 +305,55 @@ define([
     };
 
     /**
+     * Computes a 3x3 rotation matrix from the provided headingPitchRoll. (see http://en.wikipedia.org/wiki/Conversion_between_quaternions_and_Euler_angles )
+     *
+     * @param {HeadingPitchRoll} headingPitchRoll the headingPitchRoll to use.
+     * @param {Matrix3} [result] The object in which the result will be stored, if undefined a new instance will be created.
+     * @returns {Matrix3} The 3x3 rotation matrix from this headingPitchRoll.
+     */
+    Matrix3.fromHeadingPitchRoll = function(headingPitchRoll, result) {
+        //>>includeStart('debug', pragmas.debug);
+        if (!defined(headingPitchRoll)) {
+            throw new DeveloperError('headingPitchRoll is required');
+        }
+        //>>includeEnd('debug');
+        var cosTheta=Math.cos(-headingPitchRoll.pitch);
```

Comment:
Minor, but can you add whitespace here and below, e.g., `cosTheta=Math.cos` becomes `cosTheta = Math.cos`